src/components/ShareTrajectoryModal/style.css

```diff
@@ -101,3 +94,8 @@
     font-size: 19px;
     margin: 0px;
 }
+
+.embed-preview-modal :global(.ant-modal-content) {
+    width: var(--embed-preview-modal-width); 
+    height: var(--embed-preview-modal-height);
+}
```

Comment:
These CSS vars are not defined globally until the `EmbedPreview` component sets them

Reply:
these shouldn't be in this PR anymore